README.md

```
@@ -1,30 +1,253 @@
+  "ticket_id": "11111111-2222-3333-4444-555555555555",
+  "quantity": 1
+}
+```
 
 ---
 
-## 🧰 Tech Stack
+## 🧑‍🎨 Author
 
-### 🌐 Frontend
+**EventHub Project**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)